protos/sigstore_common.proto

```diff
@@ -85,9 +85,13 @@ enum PublicKeyDetails {
         PKIX_ED25519 = 7; // See RFC8032
         PKIX_ED25519_PH = 8;
 
-        // **EXPERIMENTAL**: LMS and LM-OTS variants.
-        EXPERIMENTAL_LMS_SHA256 = 14;
-        EXPERIMENTAL_LMOTS_SHA256 = 15;
+        // LMS and LM-OTS
+        //
+        // These keys and signatures may be used by private Sigstore
+        // deployments, but are not currently supported by the public
+        // good instance.
```

Comment:
Do you think a comment along the lines of "here be dragons" would be good? This would cover some of the original motivation of the experimental prefix, that when using these, you should understand the cryptographic and performance consequences.

Reply:
Good idea! I'll add some additional warning language here.e